File: WEKEE.BE/Account.Domain/Shared/DataTransfer/SubjectDTO/SubjectReadDto.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Account.Domain.Shared.DataTransfer.SubjectDTO
{
    public class SubjectReadDto
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public string UserName { get; set; }
        public int? GorupId { get; set; }
        public string GorupName { get; set; }
        public bool? IsActive { get; set; }
        public int CreateBy { get; set; }
        public string CreateByName { get; set; }
        public DateTime CreatedOnUtc { get; set; }
        public DateTime UpdatedOnUtc { get; set; }
    }
}

File: WEKEE.FE/UI.HOME/src/scenes/Admin/APermission/dto/subjectReadDto.ts
export interface SubjectReadDto {
    id: number;
    userId: number;
    userName: string;
    gorupId: number | null;
    gorupName: string;
    isActive: boolean | null;
    createBy: number;
    createByName: string;
    createdOnUtc: string;
    updatedOnUtc: string;
}